### PR TITLE
Ensured output from Correlation dialog is correct

### PR DESCRIPTION
**Pull request by Vitalis95**
Fixes partially #8104 
@lloyddewit , @Patowhiz ,  @rdstern . 
- In `multiple receiver` , I have added the following for the  new output system
`Save Table` 
When `Display Options` is checked, it has 
image

For `Save Table` and `Save data frame` when the buttons are checked respectively
- In `Two columns ` it has `Save Model` control for saving the model
- The sdg changes is because of setting R-code bug

### CODE REVIEW

**Comment by lloyddewit** [comment]
@berylwaswa please could you test? thanks

**Comment by berylwaswa** [comment]
@Vitalis95 @lloyddewit this seems to work fine.

File: instat/sdgCorrPlot.vb [hunk 1]
```diff
@@ -65,18 +75,18 @@ Public Class sdgCorrPlot
...
+        ucrPnlGraphType.AddToLinkedControls(ucrSaveGraph, {rdoPairwisePlot, rdoCorrelationPlot, rdoScatterPlotMatrix}, bNewLinkedHideIfParameterMissing:=True)
         ucrNudMinimunSize.SetLinkedDisplayControl(lblMinimumSize)
         ucrNudMaximumSize.SetLinkedDisplayControl(lblMaximumSize)
         ucrPnlGraphType.SetLinkedDisplayControl(grpOptions)
         ucrInputComboGeom.SetLinkedDisplayControl(lblGeom)
 
-        ucrPnlGraphType.AddToLinkedControls(ucrSelectorFactor, {rdoScatterPlotMatrix}, bNewLinkedAddRemoveParameter:=True, bNewLinkedHideIfParameterMissing:=True)
-        ucrPnlGraphType.AddToLinkedControls(ucrReceiverFactor, {rdoScatterPlotMatrix}, bNewLinkedAddRemoveParameter:=True, bNewLinkedHideIfParameterMissing:=True)
+        ucrPnlGraphType.AddToLinkedControls(ucrSelectorFactor, {rdoScatterPlotMatrix}, bNewLinkedHideIfParameterMissing:=True)
+        ucrPnlGraphType.AddToLinkedControls(ucrReceiverFactor, {rdoScatterPlotMatrix},, bNewLinkedHideIfParameterMissing:=True)
```
Comment (lloyddewit) [inline instat/sdgCorrPlot.vb:89]:
```suggestion
        ucrPnlGraphType.AddToLinkedControls(ucrReceiverFactor, {rdoScatterPlotMatrix}, bNewLinkedHideIfParameterMissing:=True)
```

File: instat/sdgCorrPlot.vb [hunk 2]
```diff
@@ -88,16 +98,8 @@ Public Class sdgCorrPlot
         ucrInputComboGeom.SetItems(dctGeom)
         ucrInputComboGeom.SetDropDownStyleAsNonEditable()
         ucrInputComboGeom.SetLinkedDisplayControl(lblGeom)
+        'ucrInputComboGeom.SetRDefault("tile")
```
Comment (lloyddewit) [inline instat/sdgCorrPlot.vb:101]: delete?

File: instat/sdgCorrPlot.vb [hunk 3]
```diff
@@ -193,18 +196,33 @@ Public Class sdgCorrPlot
...
+        '        clsDummyFunction.AddParameter("checked", "cor", iPosition:=0)
+
+        '    ElseIf rdoPairwisePlot.Checked Then
+        '        clsRsyntax.AddToAfterCodes(clsRGraphicsFuction, iPosition:=2)
+        '        clsDummyFunction.AddParameter("checked", "pair", iPosition:=0)
+
+        '    ElseIf rdoScatterPlotMatrix.Checked Then
+        '        clsRsyntax.AddToAfterCodes(clsRGGscatMatricReverseOperator, iPosition:=3)
+        '        clsDummyFunction.AddParameter("checked", "scatter", iPosition:=0)
+        '    End If
```
Comment (lloyddewit) [inline instat/sdgCorrPlot.vb:225]: delete?

**Review by lloyddewit — CHANGES_REQUESTED** [review]

**Review by berylwaswa — DISMISSED** [review]

File: instat/dlgCorrelation.vb [hunk 4]
```diff
@@ -481,29 +459,24 @@ Public Class dlgCorrelation
...
 
     Private Sub ChangeBaseFunction()
         If bRcodeSet Then
             If rdoMultipleColumns.Checked Then
                 If ucrChkDisplayOptions.Checked Then
-                    clsPipeOperator.RemoveAssignTo()
-                    If ucrChkDisplayAsDataFrame.Checked Then
-                        ucrSaveFashionDataFrame.SetSaveTypeAsDataFrame()
+                    If rdoAsDataFrame.Checked Then
                         ucrBase.clsRsyntax.iCallType = 0
```
[the remark this answers is not in the document]
Comment (Vitalis95) [reply instat/dlgCorrelation.vb:471]: @lloyddewit , yes it's only here

**Review by berylwaswa — COMMENTED** [review]
This looks fine.

**Review by berylwaswa — APPROVED** [review]

**Review by lloyddewit — APPROVED** [review]

**Pull request merged**